<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tracks queue</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="pages">
      <widget class="QWidget" name="page"/>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="lefttoolbar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionHome"/>
   <addaction name="actionStarred"/>
   <addaction name="actionProjects"/>
   <addaction name="actionTickler"/>
   <addaction name="separator"/>
   <addaction name="actionAdd"/>
  </widget>
  <widget class="QToolBar" name="righttoolbar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdmin"/>
  </widget>
  <action name="actionHome">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
   <property name="shortcut">
    <string>Alt+T</string>
   </property>
  </action>
  <action name="actionProjects">
   <property name="text">
    <string>Projects</string>
   </property>
   <property name="shortcut">
    <string>Alt+P</string>
   </property>
  </action>
  <action name="actionStarred">
   <property name="text">
    <string>Starred</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="actionCalendar">
   <property name="text">
    <string>Calendar</string>
   </property>
   <property name="shortcut">
    <string>Alt+L</string>
   </property>
  </action>
  <action name="actionTickler">
   <property name="text">
    <string>Tickler</string>
   </property>
   <property name="shortcut">
    <string>Alt+K</string>
   </property>
  </action>
  <action name="actionAdmin">
   <property name="text">
    <string>Admin</string>
   </property>
   <property name="toolTip">
    <string>Admin</string>
   </property>
  </action>
  <action name="actionView">
   <property name="text">
    <string>View</string>
   </property>
   <property name="toolTip">
    <string>view</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>Add new</string>
   </property>
   <property name="shortcut">
    <string>Alt+N</string>
   </property>
  </action>
  <action name="actionContexts">
   <property name="text">
    <string>Contexts</string>
   </property>
   <property name="toolTip">
    <string>Contexts</string>
   </property>
   <property name="shortcut">
    <string>Alt+C</string>
   </property>
  </action>
  <action name="actionDone">
   <property name="text">
    <string>Done</string>
   </property>
   <property name="toolTip">
    <string>Done</string>
   </property>
   <property name="shortcut">
    <string>Alt+D</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
